internal/terraform/context_plan.go

```diff
@@ -826,7 +826,7 @@ func (c *Context) planWalk(config *configs.Config, prevRunState *states.State, o
 			// In refresh-only mode we explicitly don't expect to propose any
 			// actions, but the plan is applyable if the state was changed
 			// in an interesting way by the refresh step.
-			plan.Applyable = !plan.PriorState.ManagedResourcesEqual(plan.PrevRunState)
+			plan.Applyable = !plan.PriorState.ManagedResourcesEqual(plan.PrevRunState) || !plan.PriorState.RootOutputValuesEqual(plan.PrevRunState)
```

Comment:
As noted in the PR, you could modify this to use `Equal` here. However, I opted to check specifically for root output values, as I felt that switching to `Equal` would be too big of a change.

```suggestion
			plan.Applyable = !plan.PriorState.Equal(plan.PrevRunState)
```